teacher.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Teacher < Person
+  def initialize(specialization, age, name)
+    super(age, name)
+    @specialization = specialization
+  end
```

Comment:
- [ ] Teacher constructor extends parent's constructor by adding `@specialization` and a parameter for it. One of the parent's constructor arguments is currently missing, `parent_permission = true`. Let's fix it